src/lib/utils.ts

```diff
@@ -96,3 +96,20 @@ export const getTrimmedTextContent = (
   }
   return node.querySelector(querySelector)?.textContent?.trim()
 }
+
+export const timestamp = () => Math.floor(Date.now() / 1000)
+
+export const generateAttachmentFilename = (
```

Comment:
We should get rid of the attachments code from the exporter. We can then move this under tests.

Reply:
I removed the attachment code, please let me know if we need to remove more things from there.
https://github.com/Atypon-OpenSource/manuscripts-transform/pull/123/commits/ecbb375f7729072b23b01c35938e36dccf66badd#diff-ffda4a3ab811824dfbdc9fe411843d37544de380122482986f6d3187417c68ba